src/plugins/dashboard/public/application/lib/convert_dashboard_state.ts

```diff
@@ -109,12 +116,21 @@ export const stateToDashboardContainerInput = ({
     panels,
     query,
     title,
+    filters,
```

Comment:
nit: because there is a lot of the word `filter` lower when you're checking for pinned filters, maybe you could rename this to `dashboardFilters`